lib/console.js

```diff
@@ -35,20 +37,34 @@ function Console(stdout, stderr) {
   }
 }
 
+function write(ignoreErrors, stream, string) {
+  if (!ignoreErrors) return stream.write(string);
+
+  try {
+    stream.write(string, (err) => {
+      if (err) {
+        stream.once('error', () => {});
+      }
+    });
```

Comment:
ooh, I did not notice, sorry. Add a comment there, so it's easy to understand what's going on.